src/trans.rs

```diff
@@ -282,6 +277,9 @@ impl<'v, 'tcx: 'v> BinaryenFnCtxt<'v, 'tcx> {
                     StatementKind::Assign(ref lvalue, ref rvalue) => {
                         self.trans_assignment(lvalue, rvalue, &mut binaryen_stmts);
                     }
+                    StatementKind::StorageLive(ref var) => {}
+                    StatementKind::StorageDead(ref var) => {}
```

Comment:
You can use `_` instead of `ref var` to eliminate warnings about unused variables.